lib/Yam/agama/agama_base.pm

```diff
@@ -9,19 +9,13 @@ use base 'opensusebasetest';
 use strict;
```

Comment:
For package name according to cpan we should use capitalized names, that is how we have it for our libyui-rest-api stuff. Could you please move content of `lib/Yam/agama` to `lib/Yam/Agama`, changing the `package` name above for both files?

Reply:
I think I need change these agama files's name and move to the new folder also, they depend each other.